packages/components/src/components/tabs/tab.jsx

```diff
@@ -35,6 +36,7 @@ const Tab = ({
         'dc-tabs__item--header-fit-content': header_fit_content,
         'dc-tabs__item--is-hidden': is_label_hidden,
         [`dc-tabs__item--${className}`]: className,
+        'dc-tabs--is-scrollable': is_scrollable && is_active,
```

Comment:
This is a bit smelly, as you'd expect the `dc-tabs--is-scrollable` class to be added when passing a prop of the same name (`is_scrollable`), however, there's a second condition, so something like `dc-tabs--is-scrollable-and-active` would be more appropriate here.